### PR TITLE
Adding in ability to use nginx-extras, useful with @perusio drupal-wi…

### PR DESCRIPTION
**Pull request by cferthorney**
It would be useful to add in the ability to use ```nginx_extras``` which is used with Perusio's [Nginx-with-Drupal config](https://github.com/perusio/drupal-with-nginx)

### CODE REVIEW

File: vars/Debian.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+nginx_package: nginx
```
Comment (geerlingguy) [inline vars/Debian.yml]: Having the variable in here wouldn't necessarily allow it to be overridden in a playbook/inventory. What's the use case? Is `nginx-extras` a different Nginx package entirely?

Comment (cferthorney) [inline vars/Debian.yml]: Yes it is, contains additional configurations[Nginx](https://packages.debian.org/jessie/nginx) and [nginx-extras](https://packages.debian.org/jessie/nginx-extras)

**Comment by cferthorney** [comment]
This achieves the same aims as #46.  Please choose the more appropriate fix, as I am not sure if my adjustments are of as good quality as in #46.

**Comment by geerlingguy** [comment]
This was fixed in #46; both are good options, 46 had a little better documentation so I decided to go with it instead. Thanks for the PR!

**Pull request closed without merging**